app/javascript/chat/__tests__/chat.test.jsx

```diff
@@ -226,4 +226,19 @@ describe('<Chat />', () => {
       }),
     ).toBeDefined();
   });
+
+  it('should show mention pop-up on keyUp @', async () => {
```

Comment:
Missing tests (blocking): This is great that it tests that the mention list opens when you type the `@` character, but it would also be good to have tests for a couple of other things:

-  test that the current user does not appear in the list.
- test that the list filters as the user types. For example, if your list contains nickytonline and sarthology, if I type `@n`, only `nickytonline` should be in the the list.